src/components/lib/posix/posix.c

```diff
@@ -12,9 +12,12 @@
 #include <cos_component.h>
 #include <cos_defkernel_api.h>
 #include <llprint.h>
+#if 0
```

Comment:
This is werid> Maybe just remove these includes.

Reply:
Since I'm not confident to remove all changes, this is just for testing. Will remove them in the future once we have confidence of the structure of libposix.